tests/models/store.py

```diff
@@ -233,8 +236,11 @@ def test_update_set_last_sync_revision(ru_update_set_last_sync_revision_po):
 
 @pytest.mark.django_db
 def test_update_upload_defaults(en_tutorial_po, system):
-    _update_from_upload_file(en_tutorial_po,
-                             "tests/data/po/tutorial/en/tutorial_update.po")
+    update_store(
+        en_tutorial_po,
+        [("Hello, world", "Hello, world UPDATED")],
```

Comment:
en_tutorial_po is still there and is used - eg by this test.

It still has the same unit in.

The point about updating Stores from files in tests is that previously the ``revision`` was hardcoded.

As the underlying db has changed the hardcoded revision is no longer correct and nothing gets updated etc.

These changes address this.